src/Hooks/useData.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
```

Comment:
It is standard to name your hook file to correspond with the hook your exporting. So in this case, I would name the file useCountries.js